test/ex_machina/sequence_test.exs

```diff
@@ -26,6 +26,11 @@ defmodule ExMachina.SequenceTest do
     assert 1 == Sequence.next(:month, & &1)
   end
 
+  test "can optionally set starting integer" do
+    assert "100" == Sequence.next(:dollars_in_cents, &"#{&1}", start_at: 100)
+    assert "101" == Sequence.next(:dollars_in_cents, &"#{&1}")
+  end
+
   test "let's you quickly create sequences" do
```

Comment:
What are the odds we can fix the typo in `lets` in this PR?

Reply:
Haha. Love the close look. 🔍  I fixed that in 634869754888269e11b7eeb6a36dd3231e57f7d8. Thanks for pointing it out!